test/modules_test.ts

```diff
@@ -0,0 +1,48 @@
+import PretenderES from '../src/pretender.es';
```

Comment:
Pretender imported as a ES6 module. Also not that this test file is a `ts` file.